content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -459,7 +459,7 @@ non-default field manager, as seen in the following example. The default field
 manager for kubectl server-side apply is `kubectl`.
 
 ```shell
-kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
+kubectl apply --server-side --field-manager=kubectl [--dry-run=server]
```

Comment:
I don't know if I agree, the sentence before that reads:
> As an exception, you can opt-out of this behavior by specifying a different, non-default field manager, as seen in the following example.

It says the example will use a non-default field manager (so NOT `kubectl`). Or am I misreading?

But I think stfim is making a better point, the thing is unclear in general.

Reply:
Hi @roelarents .
I agree that this section is confusing.

Can you use the sample kubectl command (`kubectl apply --server-side --field-manager=my-manager [--dry-run=server]`) to downgrade/switch from server-side apply to client-side apply?